### DISCUSSION TITLE
need help to mix USB mic audio with ATC audio

### DISCUSSION THREAD

**Discussion by kenventions** [rtl-airband/RTLSDR-Airband#493] [Q&A]
I have a USB audio dongle plugged into my Raspi and the RTLSDR-Airband audio is working great and is coming out the USB audio dongle headphone output. I have a mic plugged into the USB audio dongle mic input to pickup airplane sounds at the airport that I would like to mix with the Airband audio. The wiki mentions a mixer but I'm unsure what changes I should make to the rtl_airband.conf file to mix the two audio sources. Does anyone have an example rtl_airband.conf they could share?

**Comment by kinglevel** [comment]
[rtl_airband_mixerexample.conf](https://github.com/user-attachments/files/18271133/rtl_airband_mixerexample.conf.json)

Hello

A great idea with a second source recording the airplanes!

One of my ideas would be to have a webradio from another icecast server that i have,  that is side-chained to the airband, so when someone talks, it would lower the music.

But i believe the software as of now does not support inputs like that, only sdr inputs. (Someone fill in on this)

You can read more about the mixer here:
https://github.com/charlie-foxtrot/RTLSDR-Airband/wiki/Configuring-mixers

Attached are also an example of how i do things with the mixer.
